package/yast2-nis-client.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-nis-client
-Version:        3.1.12
+Version:        3.1.13
```

Comment:
Yes, to be consistent with other packages we should bump to 3.2.x when updating a package in `master`.